app/models/cinstance.rb

```diff
@@ -110,16 +110,15 @@ def validate_plan_is_unique?
 
   validate :same_service, on: :update, if: :plan_id_changed?
 
-  APP_ID_FORMAT = /[\w-]+/.freeze
+  APP_ID_FORMAT = /[\x20-\x7E]+/
   # letter, number, underscore (_), hyphen-minus (-), dot (.), base64 format
   # In base64 encoding, the character set is [A-Z,a-z,0-9,and + /], if rest length is less than 4, fill of '=' character.
   # ^([A-Za-z0-9+/]{4})* means the String start with 0 time or more base64 group.
   # ([A-Za-z0-9+/]{4}|[A-Za-z0-9+/]{3}=|[A-Za-z0-9+/]{2}==) means the String must end of 3 forms in [A-Za-z0-9+/]{4} or [A-Za-z0-9+/]{3}= or [A-Za-z0-9+/]{2}==
   # matches also the non 64B case with (\A[\w\-\.]+\Z)
   USER_KEY_FORMAT = /(([\w\-\.]+)|([A-Za-z0-9+\/]{4})*([A-Za-z0-9+\/]{4}|[A-Za-z0-9+\/]{3}=|[A-Za-z0-9+\/]{2}==))/
 
-  # Arbitrary limit, increase don't broke anything.
-  validates :application_id, format: { with: /\A#{APP_ID_FORMAT}\Z/ }, length: { in: 4..140 }
+  validates :application_id, format: { with: /\A#{APP_ID_FORMAT}\Z/ }, length: { in: 4..255 }
```

Comment:
Why do we have the regexp interpolated? Can't we just use 

```ruby
APP_ID_FORMAT = /\A[\x20-\x7E]\Z/
```

Reply:
sure 👍 good for me 👍 😉